site/app/templates/forum/GeneratePostList.twig

```diff
@@ -57,7 +58,7 @@
         <input type="hidden" name="display_option" value="{{ display_option }}" />
         {% include "forum/ThreadPostForm.twig" with {
             "show_post" : true,
-            "post_content_placeholder" : "Enter your reply to all here...\nPress enter for newline or ctrl+enter to post",
+            "post_content_placeholder" : "Enter your reply to all here...\nPress enter for newline or ctrl+enter to post\nor esc to exit textbox",
```

Comment:
Placeholders are visual cues only and will be ignored by most screen readers. 
Some sort of label should be used here to convey this information, especially because the escape to move to the next element is nonstandard navigation.

I think this should say "Press escape to exit textbox" if its going to be on its own line, the "or" doesn't make grammatical sense to me